src/components/AddProposalPanel.js

```diff
@@ -153,11 +155,13 @@ const AddProposalPanel = React.memo(({ onSubmit }) => {
       if (beneficiary && !isAddress(beneficiary)) {
         errors.push('Beneficiary is not a valid ethereum address')
       }
+    }
 
-      return errors
+    if (link && !FORUM_POST_REGEX.test(link)) {
```

Comment:
wondering if at some point we might want to do a check on the smart contract to avoid direct calls without it

Reply:
I don't think it will be a problem since we are recommending to use some proposal guidelines.

Reply:
but what prevents to call the contract regardless there are guidelines?

Reply:
I don't really see a reason to go trough that hurdle just to bypass the link requirement.